src/Controller/CocktailController.php

```diff
@@ -64,30 +51,14 @@ public function submitName(Request $request): Response
 
         $cocktailName->setCocktailName($cocktailNameEntity->getCocktailName());
 
+        $client = HttpClient::create();
+        $parsedData = new CocktailApiController($client);
+
         //TODO API request action
 
         return $this->render('cocktail/submit.html.twig', [
             'cocktailForm' => $form->createView(),
-            'result' => $this->cocktailNameRequest($cocktailName)
+            'result' => $parsedData->cocktailNameRequest($cocktailName)
```

Comment:
You should separate result and error variables here. As I've already mentioned in the previous PR, use DTO for transfering result and error message.